docs/1000.yaml

```diff
@@ -1733,6 +1798,9 @@ Q2022775:
 
 Q2027347:
   title: Optional stopping theorem
+  decl: MeasureTheory.submartingale_iff_expected_stoppedValue_mono
+  author: Kexing Ying
```

Comment:
```suggestion
  author: Kexing Ying, Rémy Degenne
```

Reply:
Maybe you should add your name to the file too